dist/components/SearchBox/SearchBox.js

```diff
@@ -364,7 +364,7 @@ var SearchBox = function (_React$Component) {
             { className: 'label' },
             'NYPL.ORG'
           ),
-          _react2.default.createElement(_dgxSvgIcons.RightWedgeIcon, { ariaHidden: true })
+          _react2.default.createElement(_dgxSvgIcons.RightWedgeIcon, { ariaHidden: true, focusable: false })
```

Comment:
Is the last SeachIcon supposed to get the focusable: false treatment? Line 323 in the repo code from Github.

Reply:
yes I missed one! Thanks Greg!